plugins/modem-manager/fu-mm-utils.c

```diff
@@ -151,3 +151,39 @@ fu_mm_utils_get_port_info (const gchar	 *path,
 
 	return fu_mm_utils_get_udev_port_info (dev, out_device_bus, out_device_sysfs_path, out_port_usb_ifnum, error);
 }
+
+gboolean
+fu_mm_utils_find_device_file (const gchar	*device_sysfs_path,
+			   const gchar	*subsystem,
+			   gchar		**out_device_file,
+			   GError		**error)
+{
+	GList *devices;
```

Comment:
can we use g_autolist here?

Reply:
tried to use g_autolist, but got seg_fault error. not sure why exactly, switched back to GList for now